src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -676,11 +690,61 @@ else if (t instanceof ThreadDeath)
             // TransportManager and then the TransportManager will not close.
             closeTransportManagers();
 
-            if (logger.isInfoEnabled())
+            if (includeInStatistics)
             {
-                logger.info(
-                        "Expired conference " + getID()
-                            + ". " + videobridge.getConferenceCountString());
+                Videobridge.Statistics videobridgeStatistics
+                    = getVideobridge().getStatistics();
+
+                videobridgeStatistics.totalConferences.incrementAndGet();
+
+                videobridgeStatistics.totalNoPayloadChannels.addAndGet(
+                    statistics.totalNoPayloadChannels.intValue());
+                videobridgeStatistics.totalNoTransportChannels.addAndGet(
+                    statistics.totalNoTransportChannels.intValue());
+
+                videobridgeStatistics.totalChannels.addAndGet(
+                    statistics.totalChannels.intValue());
+
+                boolean hasFailed = statistics.totalNoPayloadChannels.intValue()
+                    >= statistics.totalChannels.intValue();
+                boolean hasPartiallyFailed
+                    = statistics.totalNoPayloadChannels.intValue() != 0;
+
+                if (hasPartiallyFailed)
+                {
+                    videobridgeStatistics.totalPartiallyFailedConferences.incrementAndGet();
+                }
+
+                if (hasFailed)
+                {
+                    videobridgeStatistics.totalFailedConferences.incrementAndGet();
+                }
+
+                if (logger.isInfoEnabled())
+                {
+
+                    int[] metrics
+                        = videobridge.getConferenceChannelAndStreamCount();
+
+                    logger.info(
+                        "Expired conference id=" + getID()
+                            + ", conferenceCount="
+                            + metrics[0]
+                            + ", channelCount="
+                            + metrics[1]
+                            + ", streamCount="
```

Comment:
Can we revert to the original "video streams" here? It has special meaning (I agree that it isn't accurate) and I'd rather just use the term that we already use.